src/OSPSuite.Presentation/Presenters/Importer/ColumnMappingPresenter.cs

```diff
@@ -377,6 +377,10 @@ public IEnumerable<RowOptionDTO> GetAvailableRowsFor(ColumnMappingDTO model)
             {
                options.AddRange(metaDataCategory.ListOfValues.Keys.Select(v =>
                {
+                  metaDataCategory.ListOfImages.TryGetValue(v, out var value);
```

Comment:
I have no idea what's going on here
Why are we trying to find the image based on the value ?  Shouldn't those two lists have the same length by construction?

Reply:
I thought they should, but they do not have the same length. Actually, the ListOfImages is rarely used. So currently we check first if there is an image already for this key and if there is none, then we check the icons by the name of the organ, compartment... PeripheralVenousBlood for example has an image on ListOfImages but most of the other organs do not.

Reply:
ok.... somehow this is all surprising but clearly sthg that was there before the refactoring